operate/client/src/App/Processes/MigrationView/BottomPanel/index.test.tsx

```diff
@@ -55,11 +55,14 @@ const Wrapper = ({children}: Props) => {
   processInstanceMigrationStore.enable();
 
   useEffect(() => {
-    processInstanceMigrationStore.reset();
-    processXmlMigrationSourceStore.reset();
-    processXmlMigrationTargetStore.reset();
-    processStatisticsStore.reset();
-  });
+    return () => {
+      processInstanceMigrationStore.reset();
+      processXmlMigrationSourceStore.reset();
+      processXmlMigrationTargetStore.reset();
+      processStatisticsStore.reset();
+    };
+  }, []);
```

Comment:
This is unrelated. I just noticed it was not right. Not it is ensured that the stores are reset on unmount only.